src/core/event/createGetModifierState.js

```diff
@@ -0,0 +1,19 @@
+const modifierEventKeyMap = {
```

Comment:
Seems like this could go in a `utils.js` file together with `ElementRelativePositionTransformer.js` ?